### PR TITLE
Updated homepage with recent meetings

### PR DESCRIPTION
**Pull request by 0xpmo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -98,26 +98,23 @@ <h1>Upcoming Events</h1>
...
                             </div>
                 
                             <div class="event-body">
                                 <p>
-                                    Traditional NLP models view language as a set of fixed conventions.
-                                    Pragmatics is the study of how language is used for communicative purposes.
-                                    We explore the rational speech act (RSA) model, which is a probabilistic model
-                                    that formalizes many intuitive aspects of language and enables us to predict
-                                    human behavior.
+                                    This week we will visit the EECS 442 computer vision poster session!
```
Comment (seanstapleton) [inline index.html]: Convert this to past tense

File: index.html [hunk 2]
```diff
@@ -238,24 +235,25 @@ <h1>Recent Events</h1>
                             <div class="event-header">
                                 <!-- Title -->
                                 <div class="event-title">
-                                    <span class="event-type">Reading Group</span>
-                                    <span class="event-name"><a href="https://arxiv.org/abs/1904.01766">
-                                        VideoBERT: a Joint Model for Video and Language Learning
+                                    <span class="event-type">Tutorial Session</span>
+                                    <span class="event-name"><a href="https://docs.google.com/presentation/d/1jrlcfeAh7MynnFmvvGxQJyQ4rNSS1D3li5bC1Z86mMo/edit#slide=id.p" target="_blank">
```
Comment (seanstapleton) [inline index.html]: The tutorial repo is more relevant here, and makes it more clear who the original authors were. Let's use this link: https://gitlab.eecs.umich.edu/MSAIL/tensorflow-tutorial

(make sure to update the text below to reflect that too!)

**Review by seanstapleton — CHANGES_REQUESTED** [review]
Looks good overall just some minor changes

File: index.html (hunk 1 above)
Comment (0xpmo) [reply index.html]: made it future tense to reflect that it is the "upcoming event" since we don't technically have an upcoming event right now

**Review by seanstapleton — APPROVED** [review]
approved

**Pull request merged**